helpers/fluxwatchtower.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+echo "Checking if fluxwatchtower is installed..."
+apps_check=$(docker ps | grep "fluxwatchtower")
+
+if [[ "$apps_check" == "" ]]; then
+
+  echo "Stopping fluxwatchtower..."
+  docker stop fluxwatchtower > /dev/null 2>&1
+  sleep 2
+  echo "Removing fluxwatchtower..."
+  docker rm fluxwatchtower > /dev/null 2>&1
+  echo "Downloading containrrr/watchtower image..."
+  docker pull containrrr/watchtower:latest > /dev/null 2>&1
```

Comment:
I think we shall fork it and host it on our docker hub to have control over what exactly is being installed since its part of FluxOS now.
Since it has a prefix of flux in the name it would fall under exceptions of flux applications and same rules would apply. We shall reflect it on FluxOS UI (that this app has exception) and forbid application message broadcasting.
Furthermore we shall rename it from fluxwatchtower to flux_watchtower. This would resolve all potential application name conflicts as apps on flux can only use a-zA-Z0-9. This also simplifies/unifies other exceptions we can introduce to some special apps with the _ symbol in front.

Reply:
watchtower will not broadcast status as its not located in installed apps database